CondFormats/DataRecord/interface/HGCalMappingModuleIndexerRcd.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CondFormatsModuleIndexerRcd_HGCalMappingModuleIndexerRcd_h
+#define CondFormatsModuleIndexerRcd_HGCalMappingModuleIndexerRcd_h
+// -*- C++ -*-
+//
+// Package:     CondFormats/DataRecords
+// Class  :     HGCalMappingModuleIndexerRcd
+//
+/**\class HGCalMappingModuleIndexerRcd HGCalMappingModuleIndexerRcd.h CondFormats/DataRecords/interface/HGCalMappingModuleIndexerRcd.h
+
+ Description: [one line class summary]
+
+ Usage:
+    <usage>
+
+*/
+//
+// Author:      Pedro Vieira De Castro Ferreira Da Silva
+// Created:     Mon, 13 Nov 2023 12:02:11 GMT
+//
+
+#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+
+class HGCalMappingModuleIndexerRcd
```

Comment:
Do you foresee that the content of this record will have a different granularity than that of `HGCalMappingModuleIndexerRcd` ?
Otherwise, you don't need to have different records.